ext/pdo_firebird/tests/testdb.inc

```diff
@@ -14,15 +14,25 @@ define('PDO_FIREBIRD_TEST_DSN', getenv('PDO_FIREBIRD_TEST_DSN') ?: '');
 
 if(!PDO_FIREBIRD_TEST_DSN)
 {
-        die('Error: PDO_FIREBIRD_TEST_DSN must be set');
+	die('Error: PDO_FIREBIRD_TEST_DSN must be set');
```

Comment:
I'd stick with 4 spaces as indentation in PHP files, but I don't think we have any CS in this regard.

Reply:
OK